_posts/2014-04-02-using-modules-and-the-resolver.md

```diff
@@ -129,7 +131,7 @@ explicitly:
 
 {% highlight javascript linenos %}
 // app/helpers/trim.js
-export default function(value, options) {
+export default function([value], options) {
```

Comment:
@stefanpenner - This is for the website examples, do we want to continue to use ES5 even in gh-pages?